Adding some language for recommended stream configuration audience validation (#246)
openid/sharedsignals: openid-sharedsignals-framework-1_0.md

{: title="Create Stream Errors" #tablecreatestream}

##### Validating a Stream Creation Response

* `aud`: the Receiver SHOULD validate the `aud` in the Create Stream Response.

> A Transmitter and Receiver MAY agree upon the audience value out of band.

Reviewer: The formatting doesn't look right. Have you tried making it locally and checking the output?

Author: Looks ok? 
<img width="778" alt="Screenshot 2025-04-24 at 10 09 35 AM" src="https://github.com/user-attachments/assets/a2b62663-d3f2-4ab2-b244-76a739704537" />

> Regardless of how the audience value is agreed upon, the Receiver SHOULD ensure that it matches what is expected.

Reviewer: One nit: Can we say "matches what it expects", rather than "matches what is expected" (clarifies who sets the expectation).

> Regardless of how the audience value is agreed upon, the Receiver SHOULD ensure that it matches what is expected.

Author: ```suggestion
Regardless of how the audience value is agreed upon, the Receiver SHOULD ensure that it matches what it expects.
```

{: title="Create Stream Errors" #tablecreatestream}

##### Validating a Stream Creation Response

> * `aud`: It is RECOMMENDED that the Receiver validates the `aud` in the Create Stream Response.

Reviewer: I think you need to rephrase the sentence to use the word "SHOULD" (i.e., The Receiver SHOULD validate the 'aud' value in the Create Stream Response.

Author: ```suggestion
* `aud`: the Receiver SHOULD validate the `aud` in the Create Stream Response.
```

> In most cases, a Transmitter and Receiver will agree upon the audience value out of band. 

Reviewer: Use normative language here too. Drop the "In most cases", and say "A Transmitter and Receiver MAY agree upon the audience value out of band."

> Regardless of how the Receiver obtains the audience, it SHOULD ensure that it matches the

Reviewer: "obtains" doesn't sound right here. How about "Regardless of how the audience value is agreed upon"?
